app/letsencrypt_service

```diff
@@ -86,22 +86,29 @@ function cleanup_links {
     fi
     [[ $DEBUG == true ]] && echo "Disabled domains: ${DISABLED_DOMAINS[*]}"
 
+
     # Remove disabled domains symlinks if present.
     # Return 1 if nothing was removed and 0 otherwise.
     if [[ ${#DISABLED_DOMAINS[@]} -gt 0 ]]; then
+      echo "Some domains are disabled. Check them to remove unused symlinks."
       for disabled_domain in "${DISABLED_DOMAINS[@]}"; do
-          certfile="${disabled_domain}.crt"
+          echo -e -n "\nChecking domain ${disabled_domain}: "
+          certfile="/etc/nginx/certs/${disabled_domain}.crt"
           # If certificate is not letsencrypt, don't ever try to remove it
           if [[ -f "${certfile}" ]] && [[ -z $(openssl x509 -noout -issuer -in ${certfile} | grep "Let's Encrypt") ]]; then
+              echo "certificate is not LE. Skipping."
               continue
-          fi;
+          fi
           for extension in .crt .key .dhparam.pem .chain.pem; do
               file="${disabled_domain}${extension}"
+              echo -n -e "\nChecking ${file}"
               if [[ -n "${file// }" ]] && [[ -L "/etc/nginx/certs/${file}" ]]; then
```

Comment:
I wonder why `[[ -n "${file// }" ]]` works as expected. Could you give me a hint how to explain it?

Reply:
Wow, thank you for the detailed explanation. I rarely code something on bash, but always discover something interesting in syntax.

P.S.  First time meet developer from Nantes on GitHub :) I has a talk last year in Nantes on PHP conference PHP Tour. You have a beautiful city and wonderful Elephant)

Reply:
He he thank you, I'm relatively new in Nantes but yes that's one hell of a nice city 😃